js/msp.js

```diff
@@ -1387,6 +1388,10 @@ MSP.crunch = function (code) {
             buffer.push(lowByte(RX_CONFIG.rx_max_usec));
             buffer.push(highByte(RX_CONFIG.rx_max_usec));
             if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+                buffer.push(0); // 4 null bytes for betaflight compatibility
+                buffer.push(0);
+                buffer.push(0);
+                buffer.push(0);
```

Comment:
The 4 nulls are to reserve alignment with betaflight, so that when I add NRF24 to betaflight the MSP serialisation will be the same.